editor/src/components/inspector/inspector-strategies/fill-container-basic-strategy.tsx

```diff
@@ -29,21 +33,22 @@ export const fillContainerStrategyBasic = (
     }
 
     return elements.flatMap((path) => {
-      const parentInstance = MetadataUtils.findElementByElementPath(metadata, EP.parentPath(path))
-      if (!MetadataUtils.isFlexLayoutedContainer(parentInstance)) {
+      const instance = MetadataUtils.findElementByElementPath(metadata, path)
+      // TODO BEFORE MERGE!! this is an unrelated change I just fixed it
```

Comment:
@bkrmendy I think this one is for you. I noticed that this place was using EP.parentPath to check something and I _believe_ I replaced it with something equivalent but less error-prone. could you double check please?

Reply:
the parent instance is needed to check whether it's a flex container, what you added is in fact a safer and better way of doing that so big +1 from me